src/jsifier.js

```diff
@@ -46,6 +46,16 @@ var NEED_ALL_ASM2WASM_IMPORTS = BINARYEN_TRAP_MODE == 'js';
 // the current compilation unit.
 var HAS_MAIN = ('_main' in IMPLEMENTED_FUNCTIONS) || MAIN_MODULE || SIDE_MODULE;
 
+// Mangles the given C/JS side function name to assembly level function name (adds an underscore)
+function mangleFunctionName(f) {
```

Comment:
"mangle" might be confused with C++ name mangling. How about "asmify" (for "assembly-language-ify")? Or maybe there's something better...

I think "Function" can be removed here. It's shorter, and also this works on objects and numbers.

Reply:
Might be clearer to split out the addition and usage of the mangling helper functions into its own change?

Reply:
Might be - though I'm sure we can manage the addition in this PR as well.

I think mangling is commonly used in "C name mangling" and "C++ name mangling", I'll change to `mangleCSymbolName`